espnet2/enh/espnet_model.py

```diff
@@ -84,10 +84,14 @@ def forward(
                             espnet2/iterators/chunk_iter_factory.py
             kwargs: "utt_id" is among the input.
         """
-
-        # clean speech signal of each speaker
+        # reference speech signal of each speaker
+        assert "speech_ref1" in kwargs, "At least 1 reference signal input is required."
         speech_ref = [
-            kwargs["speech_ref{}".format(spk + 1)] for spk in range(self.num_spk)
+            kwargs.get(
+                f"speech_ref{spk + 1}",
+                torch.zeros_like(kwargs["speech_ref1"]),
+            )
+            for spk in range(self.num_spk)
```

Comment:
I see that the `num_spk` is renamed to `ref_num` in some other files (e.g. `espnet2/train/preprocessor.py`, `enh.sh`). Maybe we can also rename it here if it's compatible with the past version.

Reply:
It may need more work. Changing the name here would impact other files, like enh_inference.py and the enh_s2t tasks probably. I think the pre-trained enhancement models would be influenced, too.